lib/src/piece/block.dart

```diff
@@ -19,24 +19,19 @@ class BlockPiece extends Piece {
   /// The closing delimiter.
   final Piece rightBracket;
 
-  /// Whether the block should always split its contents.
-  ///
-  /// True for most blocks, but false for enums and blocks containing only
-  /// inline block comments.
```

Comment:
Can we still retain this comment information somehow by moving the comment somewhere with constructor?

Reply:
Oops, didn't mean to lose that. Done.